### ISSUE TITLE
Tagged release?

### ISSUE THREAD

**Issue by somasis** [asciidoc-py/asciidoc-py#15]
It'd be nice to have a tagged version of asciidoc with Python 3 support, so distro people could package it. Not sure how early it is to be asking for this, though.

**Comment by elextr** [comment]
I suspect it would be good to have affirmation from a few people who have successfully used it in practice prior to packaging, But equally it might need release to get usage.

**Comment by elextr** [comment]
Note that a lack of a tag doesn't stop distros from packaging, Fedora apparently already has packaged this, [see](https://github.com/asciidoc/asciidoc-py3/issues/16#issuecomment-396787667).

**Comment by Bluewind** [comment]
I'm packaging asciidoc for Arch Linux and we are slowly phasing out python 2. How is the state of this python 3 port nowadays? Is it stable enough that you could tag a release? If possible, I'd really like to avoid having to package some randomly selected git commit.

**Comment by MasterOdin** [comment]
The python 3 port passes all of the tests from the python 2 version with the exception of the JA test which was actually bugged in python 2 and is fixed here. Have not heard/seen anyone mention their experiences using this though beyond the issues @elextr links above.

**Comment by FranklinYu** [comment]
> I suspect it would be good to have affirmation from a few people who have successfully used it in practice prior to packaging

We can at least have [release candidate](https://en.wikipedia.org/wiki/Software_release_life_cycle#Release_candidate) if that's the concern.

**Comment by aerostitch** [comment]
asciidoc-py3 has been replacing the asciidoc package since early september in Debian too and so far we've only had 1 bug report even if it's used in a lot of automated process so it's also a pretty positive sign.
I've used it personally and had no problems with it so far.

**Comment by MasterOdin** [comment]
Yes, once #88 is merged, I plan to cut 9.0.0RC1 as I continue working on replacing the aap build system with make for the 9.0.0 final.

**Comment by FranklinYu** [comment]
@MasterOdin Great thanks for tagging the two RC! So glad to see this project still alive.

Are we closing this issue? Or close it when v9 stable is released?

**Comment by MasterOdin** [comment]
Yeah, let's close this, and I should maybe make milestones to better track what needs doing for 9.0.0 final.
